src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
+import { Props } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
+  <>
+    <ul className="movies">
+      {movies.map((movie) => (
+        <li>
+          <MovieCard {...movie} key={movie.imdbId} />
```

Comment:
```suggestion
          <MovieCard movie={movie} key={movie.imdbId} />
```

Probably it'd fix your failed tests

Reply:
Я намагався так зробити, але:
![111](https://user-images.githubusercontent.com/109183803/219908258-6f42fc26-0207-46eb-8176-03b723f339c7.jpg)